docs/source/tutorials/colab_requirements.txt

```diff
@@ -0,0 +1,213 @@
+absl-py==0.10.0
```

Comment:
@mshuaibii : Yes, ideally not using this file would be optimal. But as Elliot mentioned, there are issues with underlying linkers and some .so files are not found.

@elliottower : Would it be possible to not have an explicit requirements.txt but get this file using the yml files? I remember doing something like "conda list -e > requirements.txt"  But not sure if it is possible, so would require some digging.
If we are able to do something like this, we could avoid "!pip3 install -q -r ./ocp/docs/source/tutorials/colab_requirements.txt" and instead have the requirements file created on the fly which can be pip installed.

Reply:
That's a good idea about dynamically getting a requirements.txt from the yml files, but from my experience with it, there were a couple (5+) exceptions to the conda versions that had to be manually removed from the requirements.txt, a few of them are predictable and pertain to Jupyter, which Colab says not to mess with as it will require restarting the notebook. It would be possible to remove these exceptions but it 'd probably require some complicated unix commands like sid/awk, and might not work in the case of updated .env files. 

I think the explicit method should be fine as this is only being run in a Colab notebook (and it guarantees the versions are correct and it will run), and we couldn't get Conda working despite a lot of testing and troubleshooting.

Reply:
I did some digging and it doesn't look like there's an easy way to generate the requirements.txt dynamically. The problem is I can't get a working environment in conda without the errors with torch_sparse and so I can't export a working environment from within the Colab ecosystem, and when I copy a working environment from outside there are conflicts with Colab. The only thing I can think of potentially working is very convoluted: create a separate ocp-models env in conda (taking the required time to solve the entire environment), export the requirements.txt from that, and then install the resulting requirements.txt using pip in the base conda environment. This would effectively double the amount of time to set up the environment, and I don't imagine installing the same packages as conda already has through pip would do any help.

As a side note, if it is intended for users to re-use this or another Colab notebook for actual testing purposes, they would want to save the environment to Google Drive for it to persist between sessions (https://stackoverflow.com/questions/55253498/how-do-i-install-a-library-permanently-in-colab) and most likely also cache the data and ocp github clone to Drive as well. These are all possible but they would require the user to either sign in and authorize with Google in another tab, or manually copy their authentication key and save it in the notebook. I think it's safe to assume that people this isn't necessary for a simple test notebook though.